visualizer/service/api/v10.py

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
```

Comment:
you don't seem to be using `os` anywhere here.

Reply:
Modification made in commit 5cddb79d05f7fb24c82ab6018736cd0ab7adfd40